transformation-builder/src/main/kotlin/com/cloudinary/transformation/Transformation.kt

```diff
@@ -13,6 +16,8 @@ class Transformation(internal val actions: List<Action> = emptyList()) : ITransf
     class Builder(private val components: MutableList<Action> = mutableListOf()) : ITransformable<Builder> {
         override fun add(action: Action) = apply { components.add(action) }
         fun build() = Transformation(components)
+        fun quality(quality: Quality) = apply { components.add(Delivery.quality(quality))}
```

Comment:
Is there a better place to put these methods?

Reply:
As far as I understand we want them to be called on the transformation to shorten the syntax 
```transformation {
            quality(Quality.auto())
}
```
Which will require the functions to be under the `Transformation` Builder class